index.js

```diff
@@ -3,9 +3,10 @@ const utils = require('./helpers/utils');
 const desserts = ['akok', 'apple crumble', 'cake', 'cempedak creme', 'cempedak fritters', 'cekodok','cendol','ais kacang','eclair','chocolate','cake','tart', 'muffin', 'brownies', 'cupcake', 'frgál', 'kremrole', 'větrník', 'buchty', 'krapfen', 'kek lapis']
 const adjectives = ['bbq', 'berrylicious', 'butterscotch', 'caramel', 'cheesy', 'chocolate', 'salted egg', 'salty', 'smoked', 'strawberry', 'sugar', 'sweet']
 
-function generateDessert() {
-    const dessertIndex = utils.getRandomIndex(desserts);
-    return desserts[dessertIndex];
+function generateDessert({keyword = ''} = {}) {
```

Comment:
I intend for the dessert list to contain items that are not related to each other so even though I have cempedak creme and cempedak fritters in the list, I am considering them to be distinct and not related. I do not intend to implement this logic in my code.